jib-gradle-plugin/README.md

```diff
@@ -193,6 +193,7 @@ Field | Type | Default | Description
 `skaffold` | [`skaffold`](#skaffold-integration) | See [`skaffold`](#skaffold-integration) | Configures the internal skaffold tasks. This configuration should only be used when integrating with [`skaffold`](#skaffold-integration). |
 `containerizingMode` | `String` | `exploded` | If set to `packaged`, puts the JAR artifact built by the Gradle Java plugin into the final image. If set to `exploded` (default), containerizes individual `.class` files and resources files.
 `allowInsecureRegistries` | `boolean` | `false` | If set to true, Jib ignores HTTPS certificate errors and may fall back to HTTP as a last resort. Leaving this parameter set to `false` is strongly recommended, since HTTP communication is unencrypted and visible to others on the network, and insecure HTTPS is no better than plain HTTP. [If accessing a registry with a self-signed certificate, adding the certificate to your Java runtime's trusted keys](https://github.com/GoogleContainerTools/jib/tree/master/docs/self_sign_cert.md) may be an alternative to enabling this option.
+`configurationName` | `String` | `runtimeClasspath` | Configures the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.Configuration.html) to use. It can be used to be selective about the dependencies and artifacts added to the container.
```

Comment:
This change should be in a separate PR so that we can merge it after making the next release. 

```suggestion
`configurationName` | `String` | `runtimeClasspath` | Configures the name of the [Gradle Configuration](https://docs.gradle.org/current/dsl/org.gradle.api.artifacts.ConfigurationContainer.html) to use. It can be used to be selective about the dependencies and artifacts added to the container.
```
The new link seems more user-friendly.

Reply:
Good catch. I will move this change out of this PR.